helm/tests/storage/pvc_test.yaml

```diff
@@ -0,0 +1,73 @@
+suite: "Storage PVC Tests"
+
+templates:
+  - "templates/storage/pvc.yaml"
+
+tests:
+  - it: "should render a PVC with custom size, access mode and storage class"
+    set:
+      persistence:
+        enabled: true
+        storageData:
+          enabled: true
+          size: 10Gi
+          accessMode: ReadWriteMany
+          storageClass: sqlite-storageclass-test
+    asserts:
+      - equal:
+          path: "metadata.name"
+          value: "RELEASE-NAME-sbombastic-storage-data"
+      - equal:
+          path: "spec.storageClassName"
+          value: "sqlite-storageclass-test"
+      - equal:
+          path: "spec.accessModes"
+          value:
+          - ReadWriteMany
+      - equal:
+          path: "spec.resources.requests.storage"
+          value: "10Gi"
+  - it: "should use standard storage class when storageClass is set to '-'"
+    set:
+      persistence:
+        enabled: true
+        storageData:
+          enabled: true
+          storageClass: "-"
+    asserts:
+      - equal:
+          path: "spec.storageClassName"
+          value: "-"
```

Comment:
The description is no longer relevant, you're no longer emitting the `standard` name. 

We should check the following things:
- the name of the storageClass provided by the user is propagated. Kinda what you're doing here, but I would use valid name like `fast`.
- ensure an empty storageClass name is still propagated as empty string